src/styles.js

```diff
@@ -70,18 +65,26 @@ export default customOptions => {
     const hljsStyles = Object.keys(colors)
         .filter(k => ignoredColors.indexOf(k) === -1)
         .map(name => {
-            // convert from camelCase to hyphen-case
-            const type = name.replace(/[A-Z]/g, char => `-${char.toLowerCase()}`);
-            return `.${cssMap.container} .hljs-${type} { color: ${colors[name]}; }`;
-        }).join('\n')
-
-    insertCss(csjs.getCss(cssMap) + hljsStyles + `
-    .${cssMap.container} .hljs { background: ${colors.background}; }
-    .${cssMap.container} .hljs-emphasis { font-style: italic; }
-    .${cssMap.container} .hljs-strong { font-weight: bold; }
-    `)
+            const type = name
+                // convert from camelCase to hyphen-case
+                .replace(/[A-Z]/g, char => `-${char.toLowerCase()}`)
+                // replace stringed object keys to have no strings; i.e. {'cool-stuff': red; }
+                .replace(/['|"](.*)['|"]/g, '$1');
```

Comment:
This doesn't make sense to me; what is this supposed to be achieving?

Reply:
When the options object has a key that needs to be in quotes, `hljsStyles` wasn't picking it up. See the comment on line 71 or where I'm doing it -https://github.com/Pickra/copy-code-block/blob/bd793ff4a8851307cc37e41c5550b54077b23836/src/examples/css.stories.js#L17

Reply:
Just tested it with the above line commented out, and it works just fine.

Reply:
I don't want it commented out

Reply:
I meant without the `.replace(/['|"](.*)['|"]/g, '$1');` line.